collectors/o365/o365_collector.js

```diff
@@ -33,7 +34,7 @@ const tsPaths = [
 class O365Collector extends PawsCollector {
 
     constructor(context, creds) {
-        super(context, creds, packageJson.version);
+        super(context, creds, packageJson.version, [checkO365Subscriptions], []);
     }
     
     pawsInitCollectionState(event, callback) {
```

Comment:
I think we need to check if subscription is started in `pawsInitCollectionState()`, and start it if disabled. Otherwise if subscription is disabled we'll have to wait for 15 mins until the next checkin trigger.

Reply:
I do run the subscription at the end of this function. line 70-72.

I run the same healthcheck function. i just call it manually.